bin/run_sharpener

```diff
@@ -67,19 +65,28 @@ parser.add_argument("--do_plots", action="store_true", default=False,
 
 args = parser.parse_args()
 
+# Exit if only 
+# +++++++++++++++++
+if len(sys.argv) < 2:
+
+    print("\n\t!!! no arguments were parsed from user. Try:\n\n\trun_sharpener -h\n")
+    sys.exit(0)
+
 # Make default file
 # +++++++++++++++++
 if args.generate_default:
     time_start_load = time.time()
     from shutil import copyfile
     import sharpener
-    print("## Getting parameter file: {}".format(args.generate_default))
-    configfile = args.generate_default 
+    print("## Getting parameter file: {}".format('sharpener_default.yml'))
```

Comment:
Here are you forcing the user's default file to be named `sharpener_default.yml`?
I thought user will run `run_sharpener -gd mykoolname.yml`.